### PR TITLE
Gracefully handle leaving out the Ona or Snappy API urls.

### PR DESCRIPTION
**Pull request by hodgestar**

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: go-app.js [hunk 1]
```diff
@@ -266,19 +266,28 @@ go.app = function() {
...
-                    msisdn: self.im.user.addr,
-                    toilet: data.toilet,
-                    issue: data.issue,
-                    query: data.query
-                    //datetime: Date.now()
+            return Q()
+                .then(function() {
+                    var url = self.im.config.snappy_api_url;
+                    if (typeof url == 'undefined') {
+                        return null;
```
Comment (rudigiesler) [inline go-app.js]: Can we not log something before skipping? Just in case the config for the URL is forgotten and we don't notice.

**Comment by hodgestar** [comment]
Logging added.

**Comment by rudigiesler** [comment]
:+1:

**Pull request closed without merging**